packages/compass-e2e-tests/tests/connection.test.ts

```diff
@@ -641,6 +645,73 @@ describe('Connection string', function () {
   });
 });
 
+describe('Connect in a new window', () => {
+  let compass: Compass;
+  let browser: CompassBrowser;
+
+  before(async function () {
+    compass = await init(this.test?.fullTitle(), {
+      firstRun: false,
+    });
+    browser = compass.browser;
+    await browser.setupDefaultConnections();
+  });
+
+  after(async function () {
+    await cleanup(compass);
+  });
+
+  afterEach(async function () {
+    // Close any windows opened while testing, in reverse order
+    const [mainWindow, ...otherWindows] = await browser.getWindowHandles();
+    for (const window of otherWindows.reverse()) {
+      await browser.switchToWindow(window);
+      await browser.closeWindow();
+    }
+    await browser.switchToWindow(mainWindow);
+  });
+
+  it('can connect in new window', async function (this) {
+    skipForWeb(this, 'connecting in new window is not supported on web');
```

Comment:
Leave a TODO here with a ticket numer so that it's easier to find?

Reply:
Added a todo 👍